src/Rule/EmailHandler.php

```diff
@@ -77,13 +80,7 @@ public function validate(mixed $value, object $rule, ValidationContext $context)
         }
 
         if ($valid === false) {
-            $result->addError(
-                $rule->getMessage(),
-                [
-                    'attribute' => $context->getAttribute(),
-                    'value' => $originalValue,
```

Comment:
`value` can be useful for custom messages.

Reply:
https://github.com/yiisoft/validator/pull/352#discussion_r1006452567

Reply:
https://github.com/yiisoft/validator/pull/352#discussion_r1006471638